pages/linux/goobook.md

```diff
@@ -0,0 +1,12 @@
+# goobook
+
+> Access Google contacts from `mutt` or the command line.
+> More information: <https://manned.org/goobook>.
+
+- Allow goobook to access Google contacts using OAuth2:
```

Comment:
```suggestion
- Allow `goobook` to access Google contacts using OAuth2:
```
Although we don't recommend using the command name in command descriptions. I guess this one fine.